examples/autorun-zip-package/README.md

```diff
@@ -0,0 +1,383 @@
+2. Insert the card into a BrightSign player
+3. Power on the player
+4. The player will:
+    - Extract all files
+    - Rename `autorun.zip` to `autorun.zip.done`
+    - Reboot and run the application
+
+### 3. Verify Installation
+
+After the player reboots, you should see:
+
+-   A gradient purple background with success message
+-   Status indicator showing "Application Running"
+-   All directories (cache, config, logs, brightsign-dumps) created
+-   `autorun.zip.done` on the storage (original zip renamed)
+
+## Customizing the Package
+
+### Modify the Application
+
+Edit files in the `setup-files/` directory:
+
+-   **`autorun.brs`**: Main application logic
+-   **`index.html`**: UI and display content
+-   **`config/app-config.json`**: Application settings
+
+After making changes, rebuild:
+
+```bash
+npm run build
+```
+
+### Add Your Own Content
+
+You can add additional files and directories:
+
+```bash
+cd setup-files
+# Add your content
+mkdir -p assets/videos
+cp /path/to/your/video.mp4 assets/videos/
+# Rebuild
+cd ..
+npm run build
+```
+
+### Customize for Your CMS
+
+For CMS providers, modify the application to:
+
+-   Connect to your pairing/registration endpoint
+-   Display QR codes or pairing codes
+-   Handle device authentication
+-   Download and cache content
+
+## Partner Gallery Integration
+
+To add your autorun.zip package to the BrightSign Partner Gallery:
+
+### Requirements
+
+1. **Company Information**
+
+    - Company name (as you want it displayed)
+    - Company logo
+    - Contact email for error notifications
+
+2. **Hosted autorun.zip**
+
+    - Persistent HTTPS URL
+    - Must remain accessible and up-to-date
+    - Example: `https://yourcompany.com/partners/brightsign/autorun.zip`
+
+3. **Test Screenshot**
+    - Screenshot of the display after successful installation
+    - Shows what customers should expect to see
+
+### Testing for Partner Gallery
+
+1. **Build Your Package**
+
+    ```bash
+    npm run build
+    ```
+
+2. **Test Locally on Player**
+
+    - Copy `autorun.zip` to microSD card
+    - Insert into BrightSign player and power on
+    - Verify the application loads correctly
+    - Document what appears on screen (take screenshot)
+
+3. **Host on HTTPS**
+
+    - Upload `autorun.zip` to your web server/CDN
+    - Ensure HTTPS is enabled (required for Partner Gallery)
+    - Verify the URL is accessible: `https://yourserver.com/autorun.zip`
+
+4. **Submit to BrightSign**
+    - Provide all required information above
+    - Include your tested HTTPS URL
+    - Include screenshot from testing
+    - BrightSign will validate and add to gallery
+
+### Partner Gallery Benefits
+
+-   **Zero-touch Provisioning**: Customers can provision players directly from BSN.Cloud
+-   **Brand Visibility**: Your CMS appears in the BrightSign partner selector
+-   **Faster Onboarding**: Minimal steps from unboxing to pairing screen
+-   **Qualified Partner Status**: Join the BrightSign partner ecosystem
```

Comment:
Add, your customers and your team, gain free access to BSN.control for remote connectivity.